car.py

```diff
@@ -13,12 +13,14 @@ def accelerate(self, acceleration, time):
         acceleration ([float]): the acceleration value
         time ([time]): the time of the acceleration
 
-    """     
+    """
+    if not acceleration:
+      raise ValueError('Acceleration should not be 0.')   
     self.velocity += acceleration * time   
 
 # define a crv car.
 crv = Car()
-crv.accelerate(10, 10)
+crv.accelerate(0, 10)
```

Comment:
this will allways except.. i think you should replace the zero?

Reply:
thank you, i will remove it